elbepack/initvmaction.py

```diff
@@ -148,6 +152,54 @@ def initvm_state(self):
         return self.initvm.info()[0]
 
 
+def is_qemu_running():
```

Comment:
Why is this function needed as `test_soap_communication()` exists?

Reply:
Thanks for the hint. is_qemu_running and test_soap_communication are now aligned and optimized.

Still both functions are needed, since some usages of is_qemu_running require a quick and non-blocking check.

Reply:
I needed to change it again, since the "elbe control" command waits quite long if no communication is possible.
"is_qemu_running" is now a quick check using the socket module, which is used for positive and negative (stop command) cases.
"test_soap_communiction" is a full blown, blocking communication check with is alive feedback.